### ISSUE TITLE
Cyclone won' t load some objects in Linux

### ISSUE THREAD

**Issue by emviveros** [porres/pd-cyclone#520]
Tested in Ubuntu 12.04LTS VM like this [else github issue](https://github.com/porres/pd-else/issues/779#issuecomment-687665860). Same for Ubuntu 20.04 LTS VM.
```
cyclone/borax
... couldn't create
cyclone/decode
... couldn't create
cyclone/histo
... couldn't create
cyclone/peak
... couldn't create
cyclone/mousestate
... couldn't create
cyclone/trough
... couldn't create
cyclone/clip~
... couldn't create
```

Binaries can't be found.

All cyclone 0.5.x version fail to open it. 0.4 it's ok.

[edit]
Binaries not found:
 - borax
 - decode
 - histo
 - peak
 - mousestate
 - trough
 - clip~

**Comment by emviveros** [comment]
The process of compilation of this binaries are tested and it's all ok. 

All the missing objects are present until was uploaded to deken.

Now I will test if the problem is in deken...

**Comment by jamshark70** [comment]
What is the outcome of this issue? I could just reproduce it: [trough] can't be created. Cyclone 0.5-5; Released november 25th 2020. From deken IIRC.

Just checking. It shouldn't be closed if it was never fixed.

**Comment by porres** [comment]
it's fixed, try Cyclone 0.5-6 ;)

**Comment by jamshark70** [comment]
Great, thanks!
